conseil-common/src/main/scala/tech/cryptonomic/conseil/common/sql/CustomProfileExtension.scala

```diff
@@ -1,14 +1,17 @@
 package tech.cryptonomic.conseil.common.sql
 
-import com.github.tminglei.slickpg.ExPostgresProfile
+import com.github.tminglei.slickpg._
 import slick.basic.Capability
 import slick.jdbc.JdbcCapabilities
 
 /** Custom postgres profile for enabling `insertOrUpdateAll` */
-trait CustomProfileExtension extends ExPostgresProfile {
+trait CustomProfileExtension extends ExPostgresProfile with PgArraySupport {
   // Add back `capabilities.insertOrUpdate` to enable native `upsert` support; for postgres 9.5+
   override protected def computeCapabilities: Set[Capability] =
     super.computeCapabilities + JdbcCapabilities.insertOrUpdate
+
+  override val api = ConseilAPI
+  object ConseilAPI extends API with ArrayImplicits
```

Comment:
Couldn't we change that to `PsqlApi` or `CustomPsqlApi`? First time I saw it, I thought we are talking about API that is responsible for exposing the data.

Reply:
outdated